.github/go-coding-style.md

```diff
@@ -161,7 +164,7 @@ func split(sum int) (x, y int) {
 ```
 
 ```go
-// good: it is known what is being returned by looking only at the return statement
+// Good: it is clear what is returned by looking only at the return statement
 func split(sum int) (x, y int) {
```

Comment:
I'm not sure about named return values and especially the fact that they show up in an example labeled as "good". In https://github.com/stackrox/stackrox/pull/7562#discussion_r1303108087 are you saying that this code snippet is reasonable?
```
func split(sum int) (x, y int) {
	if x % 2 == 0 {
		return x, x * x
	}
	x = sum * 4 / 9
	y = sum - x
	return x, y
}

Reply:
Well, naked returns are only possible when using named return arguments. I personally do not have anything against named arguments for return and find your example fully okay, but I see in which direction you are pulling.

Are you saying that we should avoid named return arguments all the time, or this example isn't great to illustrate the problem of naked returns?

Reply:
>  before we had a clear statement "no naked returns".  Note that this statement says nothing about named return args.

I understand, but naked returns are possible only when using named args. I deducted from that, that named args are okay but not with naked returns.
It is good that we started this discussion as we will have more clarity now as to what shall be used and what not.

Reply:
Return parameters names may add additional hint to what is actually returned, if that's not possible/desirable to express in the function name.
Consider:
```go
type Figure interface {
	GetCenter() (x float, y float)
}
```

Reply:
Perfect. Indeed, glad we spoke. Let me know once you make the change and I'll approve the PR.